Formula/sfk.rb

```diff
@@ -1,9 +1,9 @@
 class Sfk < Formula
   desc "Command-line tools collection"
   homepage "http://stahlworks.com/dev/swiss-file-knife.html"
-  url "https://downloads.sourceforge.net/project/swissfileknife/1-swissfileknife/1.9.2.0/sfk-1.9.2.tar.gz"
-  version "1.9.2.0"
-  sha256 "4537d033edfe6feb0bbafdcf157646a19951b8ef9056898718674d39a3c088d7"
+  url "https://downloads.sourceforge.net/project/swissfileknife/1-swissfileknife/1.9.4.3/sfk-1.9.4.tar.gz"
+  version "1.9.4.3"
```

Comment:
Does the version not get detected correctly without this? Can't this be removed?

Reply:
I'm pretty sure the version gets detected as `1.9.4`, which might make future updates more difficult (e.g. if `1.9.4.4` comes out).